src/test/java/com/teragrep/pth_06/planner/walker/conditions/ElementConditionTest.java

```diff
@@ -84,20 +83,6 @@ void testStreamTags() {
         Assertions.assertEquals(loops, streamTags.length);
     }
 
-    @Test
-    void testIndexStatement() {
-        Element element = document.createElement("indexstatement");
```

Comment:
what happened to this test?

Reply:
I moved indexstatement testing to IndexStatementConditionTests but I added a testIndexStatementWithBadConnection to ElementConditionTest that checks for DataAccessException with a bad connection. Also added same check for IndexStatementConditionTest.